yaml/buildings.yaml

```diff
@@ -19,7 +19,9 @@ globals:
       building_levels: {col: "building:levels"}
       building_min_levels: {col: "building:min_levels"}
       building_material: {col: "building:material"}
+      castle_type: {col: castle_type}
```

Comment:
This would be better to put into a kind_detail value, with an allowlist.

Reply:
Yes, sure, I can do it. But I am not sure how. It is a very long time when I was changing it, so I cannot remember how this could be done. Thank you.